src/sass/_cluster_specific_input.scss

```
@@ -8,4 +8,8 @@
   .cluster-specific-member {
     overflow: hidden;
   }
+
+  .clustered-input-helptext {
```

```suggestion
  .cluster-specific-helptext {
```
Better keep the naming aligned here and in the usage